dashboard/test/controllers/api/v1/pd/application/teacher_applications_controller_test.rb

```diff
@@ -33,17 +33,19 @@ class TeacherApplicationsControllerTest < ::ActionController::TestCase
     test_user_gets_response_for :create, user: :student, params: -> {@test_params}, response: :forbidden
     test_user_gets_response_for :create, user: :teacher, params: -> {@test_params}, response: :success
 
-    test_user_gets_response_for :send_principal_approval,
-      name: 'program managers can send_principal_approval for applications they own',
-      user: -> {@program_manager},
-      params: -> {{id: @application.id}},
-      response: :success
-
-    test_user_gets_response_for :send_principal_approval,
-      name: 'program managers can not send_principal_approval for applications they do not own',
-      user: :program_manager,
-      params: -> {{id: @application.id}},
-      response: :forbidden
+    puts "skipped: Implement Principal Approval 2021"
```

Comment:
I think dashboard unit tests usually leave the code intact but use `skip`.

Reply:
Oh, maybe this is different?

Reply:
yeah, i couldn't figure out the syntax for doing that in this test, but I'm already working on the PR to fix the test so it won't be forgotten.